### ISSUE TITLE
Error loading images with jest and detox

### ISSUE THREAD

**Issue by SMJ93** [wix/detox#263]
#### Description

We have installed detox and have set it up to run with Jest using help from [this](https://github.com/wix/detox/issues/143) issue.

The app runs fine through `xcode` and when running `react-native run-ios` through the command line, but when running through detox we get the following error:

```
Missing request token for request:
<NSMutableURLRequest:
0x61000000d980> { URL: http://localhost:8081/assets/node_modules/interface/assets/header.png?platform=ios&hash=######HASH######
```

There seems to a problem loading assets with detox. If I reload the app it loads as expected.

#### Steps to Reproduce

Setup detox with Jest using the issue above. Run a React Native app which uses images.

* Node: 8.1.0
* Device: iPhone 6 10.3
* Xcode: 9 beta
* macOS: Sierra 10.12.6
* React Native version: 0.47.2

**Comment by LeoNatan** [comment]
Could this be related to this?
https://github.com/facebook/react-native/issues/10553

The comments there are not encouraging. 
>same on RN 0.46.4, random happen.

>random happen.reload again

>This issue is occurring for me as well.

**Comment by SMJ93** [comment]
Yeah I saw that, but it's weird how this issue only occurs when running detox tests. We've never had this problem running the app on Android or iOS.

**Comment by LeoNatan** [comment]
I am not really an RN guy, so I cannot really help here. Just tried to look for the issue. In that FB issue, there is no real explanation why it happens so that I could think what could be happening that detox might be doing. I don’t see detox in any of those flows, so I doubt it has anything to do with this issue, but I will keep the issue open for now until there is a clearer explanation what’s going on.

If you can provide a demo project where this reproduces, please attach it here.

**Comment by SMJ93** [comment]
We fixed this issue by updating the detox configuration to run in `Release` mode in the `package.json`:
```
"detox": {
  "configurations": {
    "ios.sim.release": {
      "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/Devlyfe.app",
      "build": "xcodebuild -project ios/homelyfeMobile.xcodeproj -scheme Devlyfe -configuration Release -sdk iphonesimulator -derivedDataPath ios/build",
      "type": "ios.simulator",
      "name": "iPhone 6"
    }
  }
},
```

**Comment by Obooman** [comment]
update configuration from debug to release is totally not a solution, rebuild a release app is painful.Any other suggestions?

**Comment by SMJ93** [comment]
@Obooman it seems to be a problem with Jest on iOS. The solution maybe mocking images.

We managed to get both iOS and Android running with mocha and `Detox 5.9.1`.

I've tried using Jest as a test runner - see [here](https://github.com/wix/detox/blob/master/docs/Guide.Jest.md), but have encountered a few problems compared to using it with Mocha #362.

I will try to investigate further and let you know if I find a fix.
